src/basics.md

```diff
@@ -412,40 +412,38 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
-[error-handling]: #error-handling
-<a name="error-handling"></a>
+[ex-simple-error-handling]: #ex-simple-error-handling
+<a name="ex-simple-error-handling"></a>
 ## Handle errors correctly in main
 
 [![error-chain-badge]][error-chain] [![cat-rust-patterns-badge]][cat-rust-patterns]
 
-Uses [error-chain] to handle error occurred when trying to open a file that does not exist.
+Handles the error that occur when we try to open a file that does not exist. We do this using [error-chain], a library that takes care of a lot of boilerplate code that an user would have to write in order to handle erros in Rust. For more details please check the chapter [Error Handling] of TRPL.
 
-First, we call `error_chain!{}` inside a new module named  _errors_. This will create the default types `Error`, `ErrorKind`, `ResultExt` and ` Result`, and store them in `errors`. To be able to use these types we must call `use errors::*`.
+The [`error_chain!`] macro creates the types [`Error`], [`ErrorKind`], [`Result`] and [`ResultExt`]. Additionaly, we use `Io(sd::io::Error)` for automatic conversion between this error chain and other error types not defined by [`error_chain!`]. In this case, [`std::io::Error`]
```

Comment:
I would avoid going into details of what goes behind the scenes. The reader interested in the internals will go into linked docs. We should keep the noise to minimum with this basic example. Please remove the mention  of " [`Error`], [`ErrorKind`], [`Result`] and [`ResultExt`]. and their links".

Also please see if you can make the description a little shorter